### ISSUE TITLE
Problem loading a data-uri after updating to 2.0.0

### ISSUE THREAD

**Issue by jessegrosjean** [webpack/less-loader#31]
Likely user error, but I can't seem to figure it out...

I just updated a number of packages, including updating the less-loader to version 2.0.0. In the process of doing those updates using a `data-uri` in my less file seems to have broke:

```css
background-image: data-uri('image/svg+xml;charset=UTF-8', 'dropIndicator.svg');
```

When I do that I get this error message:

```
ERROR in ./~/css-loader!./~/less-loader!./src/birch/style/style.less
Module build failed: error evaluating function `data-uri`: Cannot 'resolveSync' because the fileSystem is not sync. Use 'resolve'!
 @ /Users/jesse/Dropbox/Projects:Hacks/Birch/src/birch/style/style.less (line 267, column 19)
 near lines:
   	content: "";
   	background-image: data-uri('image/svg+xml;charset=UTF-8', 'dropIndicator.svg');
   	background-repeat: no-repeat;
 @ ./src/birch/style/style.less 4:14-246
```

Can someone tell me what must be done to fix the problem? I'm not sure what is meant by the "Use 'resolve'!" suggestion in the error message.

Thanks,
Jesse

**Comment by jhnns** [comment]
That's an issue with less (and has been fixed already)

See less/less.js#2360

**Comment by sethyuan** [comment]
How is it fixed? I don't see any relevancy there.

I'm having the exact problem, with the latest less 2.4.0 and the latest less-loader 2.0.0

**Comment by mdreizin** [comment]
@jhnns Unfortunately, it doesn't work for now.

**Comment by mdreizin** [comment]
@jhnns A quick note: `data-uri(<resource>)` will work if `<resource>` is defined under `entry` section of `webpack.config.js`

**Comment by jhnns** [comment]
I'll take a look

**Comment by joscha** [comment]
it only seems to happen when starting the webpack dev server from scratch. if the less file is already loaded and a `url(...)` statement is changed into `data-uri()` it seems to work...

**Comment by joscha** [comment]
the workaround with the `url-loader` in #38 seems to work for this problem as well...

**Comment by jhnns** [comment]
Fix shipped with `2.2.2`
